SwanCustomEnvironments/swancustomenvironments/scripts/builders/accpy.sh

```diff
@@ -24,14 +24,17 @@ if [ "${RESOLVED_REQ}" = true ]; then
     # Use the same pip configuration as the Acc-Py default pip
     ACCPY_PIP_CONF="-i $(pip config get global.index-url) --allow-insecure-host $(pip config get global.trusted-host)"
     uv pip install ${ACCPY_PIP_CONF} -r "${REQ_PATH}" 2>&1
+    if [ $? -ne 0 ]; then
+        return 1
```

Comment:
Why is it return 1 and not exit 1? exit 1 will make the source command exit with a value different than zero, no?